controllers/storagecluster/cephcluster.go

```diff
@@ -203,6 +205,17 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		cephCluster.Spec.HealthCheck.DaemonHealth.Monitor.Timeout = "15m"
 	}
 
+	IPFamily, isDualStack, err := getIPFamilyConfig(r.Client)
+	if err != nil {
+		r.Log.Error(err, "failed to get IPFamily of the cluster")
+		return reconcile.Result{}, err
+	}
+
+	if IPFamily != "" {
```

Comment:
When the var IPFamily is empty we don't need to set the vars on the cephcluster at all?

Reply:
That's right. IPFamily empty would mean that is a normal IPv4 cluster that we use currently. We don't need to set anything there.